src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -24,7 +24,7 @@ nginx_sites:
    file_name: "{{ deploy_env }}_commcare"
    listen: "443 ssl http2{{ ' default_server' if deploy_env == primary_ssl_env else '' }}"
    root: "/var/www/html"
-   server_name: "{{ SITE_HOST }}"
+   server_name: "{{ SITE_HOST }} {{ ALTERNATE_HOSTS }}"
```

Comment:
oh wow, this is sufficient for nginx to direct requests properly?  That's fantastic.

Reply:
Yes Ethan